src/main/java/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.FileReaderService;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private String filePath;
+
+    public FileReaderServiceImpl(String filePath) {
+        this.filePath = filePath;
+    }
```

Comment:
add filePath as the parameter of the method` readFromFile(String filePath)`
```suggestion
    public FileReaderServiceImpl() {
    }
```